### ISSUE TITLE
Setting of default domain correctly

### ISSUE THREAD

**Issue by utherp0** [RedHatEMEA/demobuilder#45]
Is it possible to get the installation to default the domain on application creation to xxxx.cloudapps.example.com - currently it defaults to openshiftapps.com which if you forget to change it makes the service unreachable.

Is this a config option in OSE3 install?

**Comment by edseymour** [comment]
It's not an option during the installation, but it is something that can be configured in the master-config.yaml file. 
However, this should be done before the templates are loaded - and may even require some tinkering with templates after the import (this currently isn't clear).

**Comment by jim-minter** [comment]
@utherp0 what are you asking for?

Regardless of the domain name of the route, it should be accessible from the VM because the VM should automagically add it to its /etc/hosts for you.  Is this not happening?

Upstream, the templates are gradually being fixed to honour the installation-wide suffix which as Ed says can be set in master-config, but this hasn't been done across the board yet.

**Comment by utherp0** [comment]
Nope, if I manually change the domain to xxxx.cloudapps.example.com it works, anything else gives a 503 error at the moment.

**Comment by jim-minter** [comment]
@utherp0 got time for a bluejeans session so I can see?

**Comment by utherp0** [comment]
Sorry, not at the mo - system is now breaking all builds and I've just had an urgent 175 point OSE RFI dropped on me which I'm just plowing through :-( Might have to put the OSE3 stuff on hold until Tues.

**Comment by utherp0** [comment]
Actually, I'll setup a BlueJeans, RFI is a couple of days work at least

**Comment by jim-minter** [comment]
looks like dns-intercept got wedged

**Comment by jim-minter** [comment]
should be fixed in 2cf3b9b6c84e15a64e71996bba36ac63fb3ca235 , await rebuild
